src/powersoftau_export_json.js

```diff
@@ -17,19 +17,20 @@
     along with snarkJS. If not, see <https://www.gnu.org/licenses/>.
 */
 
-import * as utils from "./powersoftau_utils.js";
+import * as potUtils from "./powersoftau_utils.js";
 import * as binFileUtils from "@iden3/binfileutils";
-import { stringifyBigIntsWithField } from "./misc.js";
+import { readG1, readG2 } from "./misc.js";
+import { utils as ffUtils} from "ffjavascript";
 
 export default async function exportJson(pTauFilename, verbose) {
     const {fd, sections} = await binFileUtils.readBinFile(pTauFilename, "ptau", 1);
 
-    const {curve, power} = await utils.readPTauHeader(fd, sections);
+    const {curve, power} = await potUtils.readPTauHeader(fd, sections);
```

Comment:
If we don't need to import ffjavascript utils due to using the `stringifyBigIntsWithField` function in `./misc,js`, we should just keep `potUtils` named as utils so we can reduce the diff

Reply:
How strongly do you feel about this?  :).

The main reason why I renamed it is to make the code slightly more easier to read since there are multiple util imports from different files (powersoftau_utils, binfileutils, and ffjavascript).

So the rename to `potUtils` will make it a bit more explicit of which utils its referring to.

Reply:
I was trying to keep the diff absolute minimal so we can send this to Jordi. I think a refactor PR with those renames throughout would also be valuable. Should we split the changes?